examples/http_async_server.hpp

```diff
@@ -67,13 +71,138 @@ class http_async_server
             t.join();
     }
 
+    void
+    log_args()
+    {
+    }
+
+    template<class Arg, class... Args>
+    void
+    log_args(Arg const& arg, Args const&... args)
```

Comment:
I didn't really pay it any mind since its example code and not a public interface, but I can change it.